internal/command/versions.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/cyberark/summon/pkg/summon"
+	"github.com/cyberark/summon/provider"
+)
+//Create the AllVersions string to include versions of all the providers
+const SUMMON_VERSION = summon.VERSION
+var DefaultPath = provider.DefaultPath
+var AllVersions = GetVersions(GetProviderFileNames())
+
+//Run `$provider --version`(from DefaultPath),
+// 	for each provider in the list of providers
+//Then appends the responded versions to string and returns it
+func GetVersions(providers []string) string{
+	var toReturn = SUMMON_VERSION
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(DefaultPath, provider), "--version").Output()
+		if err != nil{
+			toReturn = toReturn + "\n"+provider+" version "+"ERROR"
+			fmt.Println("ERROR: "+provider, err, "running command :"+DefaultPath+"/"+ provider, "--version")	//debug
+			continue
+		}
+		toReturn = toReturn + "\n"+provider+" version "+string(version[0:len(version)-1])
+	}
+	return toReturn
+}
+
+//Create slice of all file names in the default path
+func GetProviderFileNames() []string{
```

Comment:
As discussed before, this should be done through the existing logic where possible

Reply:
Also, if you are not going to need this method public, it should not start capitalized.